src/components/ToDoDashboard/ToDoDashboard.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import AddToDo from '../AddToDo/AddToDo';
+import ToDoItem from '../ToDoItem/ToDoItem';
+import { ToDoContext } from '../../contexts/ToDoProvider';
+import { useToDo } from '../../hooks/UseToDo/UseToDo';
+
+function ToDoDashboard(): JSX.Element {
+  const { toDoList, setToDoList } = useContext(ToDoContext);
```

Comment:
You don't need context here. You have to get everything you need from a custom hook.